src/packages/commands/rat_command.py

```diff
@@ -43,7 +43,37 @@ class NameCollisionException(CommandException):
 _registered_commands = {}  # pylint: disable=invalid-name
 
 # character/s that must prefix a message for it to be parsed as a command.
-PREFIX = config['commands']['prefix']
+PREFIX: str
```

Comment:
[Question] If prefix handling is done in Context, why does RatCommand need its own?

Reply:
curiously enough, its used (outside configuration validation) in exactly one place within the module, and it doesn't even need to be there.

```py
                LOG.debug(f"Could not find command or rule for {PREFIX}{ctx.words[0]}.")
```

Il consider removing it.